### PR TITLE
getCity and getCountry now feed into getLocation and return the result

### PR DESCRIPTION
**Pull request by gleason9113**
Changes
-getCityData and getCountryData now redirect to getLocationData and return the result - calling getLocationData directly is the preferred method, but if there is code that depends on those two functions, they still work.
-The handlesearchclick in MainView now calls the appropriate function.  Currently this just prints to console, bc I'm not sure what to do with it.

### CODE REVIEW

File: dashboard/src/api.js [hunk 1]
```diff
@@ -199,7 +163,7 @@ export async function getLocationData(location) {
 
 export async function getCoords(cityName) {
   try {
-    const url = `https://api.positionstack.com/v1/forward?access_key=${AN_API_KEY}&query=${encodeURIComponent(cityName)}`;
+    const url = `http://api.positionstack.com/v1/forward?access_key=${AN_API_KEY}&query=${encodeURIComponent(cityName)}`;
```
Comment (CoreyPatrickPDX) [inline dashboard/src/api.js:166]: I'm guessing this is needed to be able to call the API?

**Review by CoreyPatrickPDX — COMMENTED** [review]
Looks good!

**Comment by CoreyPatrickPDX** [comment]
Thinking about it, we may be able to just replace all uses of getCountry or getCity with getLocationData

File: dashboard/src/api.js (hunk 1 above)
Comment (gleason9113) [reply dashboard/src/api.js:166]: Https is a premium service under this API :D

**Pull request merged**